Code/Framework/AzCore/AzCore/Console/ConsoleFunctor.cpp

```diff
@@ -172,4 +172,22 @@ namespace AZ
     {
         return GetValueResult::NotImplemented;
     }
+
+#if defined(CARBONATED)
+    bool ConsoleFunctorBase::PerformCommand(CVarFixedString inString)
+    {
+        auto console = m_console;
+        if (!console)
+        {
+            console = AZ::Interface<IConsole>::Get();
+        }
+        if (console)
+        {
+            auto res =
+                m_console->PerformCommand(AZStd::string::format("%s %s", m_name, inString.c_str()).c_str(), ConsoleSilentMode::Silent);
```

Comment:
Should this line be console->Perform command? That first if implies m_console could be null, unless getting the iconsole sets that somehow as a sideeffect